spiceypy/spiceypy.py

```diff
@@ -74,6 +75,37 @@
 _SPICE_EK_MAXQSEL = 100  # Twice the 50 in gcc-linux-64
 _SPICE_EK_EKRCEX_ROOM_DEFAULT = 100  # Enough?
 
+import threading
+
+_spicelock = threading.RLock()
+
+enablethreadinglock = False
```

Comment:
move to https://github.com/AndrewAnnex/SpiceyPy/blob/main/spiceypy/config.py

Reply:
moved